clients/src/main/java/org/apache/kafka/common/requests/DescribeLogDirsResponse.java

```diff
@@ -52,6 +52,7 @@ public int throttleTimeMs() {
     @Override
     public Map<Errors, Integer> errorCounts() {
         Map<Errors, Integer> errorCounts = new HashMap<>();
+        errorCounts.put(Errors.forCode(data.errorCode()), 1);
```

Comment:
Do we need to cover it in `testErrorCountsIncludesNone`?

Reply:
Yes, good point. I've pushed an update to cover it.